scripts/ExportFromMikePlus.py

```diff
@@ -0,0 +1,71 @@
+"""
+This is an example script showing how to export
+from Mike+ setup file (mupp/slite) to Mike1D m1dx file.
+"""
+
+import clr
+import sys
+
+clr.AddReference("DHI.Mike.Install, Version=1.0.0.0, Culture=neutral, PublicKeyToken=c513450b5d0bf0bf")
+from DHI.Mike.Install import MikeImport, MikeProducts
+
+
+def SetupMikePlusInstallation():
+    products = list(MikeImport.InstalledProducts())
```

Comment:
Does it work to instead do:
```
MikeImport.SetupLatest(MikeProducts.MikePlus)
```
?

Reply:
It just tried a few environments:
1. Python 3.6 and pythonnet 2.3.0 (does not work)
2. Python 3.8 and pythonnet 2.5.1 (does work)
3. IronPython 2.7.9 (does work)

The annoying thing with newer version of pythonnet (>2.3.0)  is that it makes implicit conversions of C# List to Python list:
https://github.com/pythonnet/pythonnet/pull/963

Still, I will use your suggestion.